_rules/motion-actuation-bf0de1.md

```diff
@@ -0,0 +1,1459 @@
+---
+id: bf0de1
+name: No functions are exclusively operated through motion actuation
+rule_type: composite
+description: |
+  This rule checks that functionality that can be operated by device motion or user motion can also be operated by user interface components and responding to the motion can be disabled to prevent accidental actuation.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_rules:
+ - c249d5
+ - 7677a9
+ - ed12b1
+ - 9d42d6
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] that has [functionality][] that can be operated by [device motion][] or user motion.
+
+## Expectation (1)
+
+For the test target, [device motion][] and user motion [can be disabled][] and the control for disabling them must be a [user interface component][] and it should be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+## Expectation (2)
+
+For the test target, [device motion][] and user motion can also be operated by [user interface components][] and they should be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
```

Comment:
Expectations for Composite rules can only look at the outcomes of each individual atomic rule. See, for example, https://act-rules.github.io/rules/80af7b#expectation

---
Additionally, in that case, I believe that the Expectation will be "the outcome of each of these rules is either Pass or Inapplicable". This is supported by the fact that each individual atomic rule is mapping to the SC (failing a single atomic rule is failing the SC).

This means that the composite rule **is not needed at all**! We already know that failing a single atomic rule is failing the SC, the composite rule is currently not giving any additional knowledge and is just useless.